charts/opensrp-web/values.yaml

```diff
@@ -186,6 +187,21 @@ express:
   sessionPath: "/"
   reactBuildPath: "/usr/src/web"
   nodeEnv: "production"
+  csp:
+    report-uri: "https://{{ .Values.sentry.domain }}/api/{{ .Values.sentry.projectId }}/security/?sentry_key={{ .Values.sentry.sentryKey }}&sentry_environment={{ .Values.sentry.environment }}&sentry_release={{ .Release.Name }}"
```

Comment:
also add `report-to` as `report-uri` is being deprecated. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri